lib/ae_mdw/db/sync/name.ex

```diff
@@ -45,6 +47,11 @@ defmodule AeMdw.Db.Sync.Name do
         previous = ok_nil(cache_through_read(Model.InactiveName, plain_name))
         expire = height + :aec_governance.name_claim_max_expiration(proto_vsn)
 
+        if expire != node_name_ttl(plain_name, height + 1) do
```

Comment:
first difference caught: `16:59:15.984 [error] [4054] name expire differ for bear.test`
```
iex(aeternity@localhost)8> expire = height + :aec_governance.name_claim_max_expiration(Util.proto_vsn(4055))
54054
iex(aeternity@localhost)9> Name.node_name_ttl("bear.test", 4055)                                            
40054
```

Reply:
54054 is correct. Another micro-block afterwards in the same height changes the ttl to 40054.